Logger.md

```diff
@@ -0,0 +1,119 @@
+# Logger
+
+`Logger` is an interface that helps decouple client classes from concrete implementations of a logging system.
+
+## Declaration
+
+```swift
+public enum LogLevel {
+    case info
+    case warning
+    case error
+}
+
+public protocol Logger {
+  func log(_ level: LogLevel, tag: String, message: String)
```

Comment:
@newboadki here also! level defaults to .info, tag must be an optional string defaulted to nil, and message should have a blank parameter name `_ message: String`

Reply:
Protocols declarations do not allow default values.

Reply:
this is true. My bad. However, you can still put `_ message: String`

Reply:
True. my bad. Nothing to be done then.